tests/sentry/audit_log/test_manager.py

```diff
@@ -0,0 +1,51 @@
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.audit_log import AuditLogEvent, audit_log_manager
+from sentry.models import AuditLogEntry, AuditLogEntryEvent
+from sentry.testutils import TestCase
+
+
+class AuditLogEventManagerTest(TestCase):
+    def test_audit_log_manager(self):
+        log_event = AuditLogEvent(
+            event_id=1,
+            name="member_invite",
+            api_name="member.invite",
+            template="invited member {email}",
+        )
+
+        log_entry = AuditLogEntry.objects.create(
+            organization=self.organization,
+            event=AuditLogEntryEvent.MEMBER_INVITE,
+            actor=self.user,
+            datetime=timezone.now(),
+            data={"email": "my_email@mail.com"},
+        )
+
+        audit_log_manager.add(log_event)
+
+        assert audit_log_manager.get(event_id=1) == log_event
+        assert audit_log_manager.get_event_id(name="member_invite") == 1
+        assert audit_log_manager.get_api_names() == ["member.invite"]
```

Comment:
Outside of testing do you see these methods being used often?

Reply:
@markstory The `get_event_id` method will be used frequently when creating new audit logs. 
The `get_api_names` method will be used to serve that data to the frontend for the audit log's drop down filter, and using it will make extending the audit log to getsentry easier since we won't need a frontend component for the dropdown filter in getsentry.